spec/features/cfp_ability_spec.rb

```diff
@@ -75,7 +75,7 @@
       # Event, booth, track cfps exist
       call_for_tracks = create(:cfp, cfp_type: 'tracks', program: conference.program)
       visit new_admin_conference_program_cfp_path(conference.short_title)
-      expect(current_path).to eq root_path
+      expect(current_path).to eq conference_path(conference)
```

Comment:
We should not change all the tests. The root_path is the default path when CanCanCan's authorization fails. I would create another conference, so that we don't fall into the exception.

When reading these tests, which are about abilities, thus about CanCanCan, it makes more sense to read root_path